pinot-common/src/main/java/com/linkedin/pinot/common/data/DataManager.java

```diff
@@ -15,44 +15,54 @@
  */
 package com.linkedin.pinot.common.data;
 
-import com.linkedin.pinot.common.config.TableConfig;
+import com.linkedin.pinot.common.metrics.ServerMetrics;
 import com.linkedin.pinot.common.segment.SegmentMetadata;
+import java.io.File;
 import java.util.List;
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 import org.apache.commons.configuration.Configuration;
+import org.apache.helix.ZNRecord;
+import org.apache.helix.store.zk.ZkHelixPropertyStore;
 
 
+// TODO: maybe better to merge APIs for adding segment to OFFLINE/REALTIME table
```

Comment:
Since FileBasedDataManager is now removed, can we just pass in serverConf to the init method? We will need to get rid of DataManager, InstanceDataManager interfaces, but that should be OK since there are no other DataManagers (and none in sight)

Reply:
We need to move huge number of files to solve this because a lot of classes/interfaces in pinot-common / pinot-core use these 2 interfaces.

Reply:
ok, agreed.